### PR TITLE
Change default whats new page banner

### LINKED ISSUE #8108: Default whatsnew page top banner is solid white
## Description

The banner on the top of the default whatsnew page, `firefox/whatsnew/index.html` is solid white and should be changed to `$color-off-black` to match the background.

Screen Shot 2019-11-08 at 9 12 49 AM

Screen Shot 2019-11-08 at 9 13 04 AM

### PR DESCRIPTION
**Pull request by imskr**
## Description
Default whatsnew page top banner is solid white
## Issue / Bugzilla link
Fixes #8108

### CODE REVIEW

File: media/css/firefox/whatsnew/whatsnew.scss [hunk 1]
```diff
@@ -61,7 +61,6 @@ $image-path: '/media/protocol/img';
 // Main content section
 
 .main-content {
-    background: $color-off-black url('/media/img/firefox/whatsnew/whatsnew_background.svg') no-repeat;
     @include background-size(100%);
```
Comment (amychurchwell) [inline media/css/firefox/whatsnew/whatsnew.scss]: We should also remove the `@include background-size(100%);` from this selector.

Comment (imskr) [reply media/css/firefox/whatsnew/whatsnew.scss]: Oh I missed it!

**Comment by imskr** [comment]
@amychurchwell Done!

**Comment by imskr** [comment]
I'm happy to help @amychurchwell 😁

**Pull request merged**